content/outcomes/application-development/cloud-native-development.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Cloud Native Development
+weight: 40
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+
+Cloud Native is at the heart of modern application development endeavors. Used to depict engineering practices and state of the art applications, the term "cloud-native" is omnipresent and often misunderstood. There is a good chance that anyone you ask will provide a different definition.
+
+![Cloud Native Development](/images/outcomes/app-dev-onboarding/cloud-native-development.jpg)
```

Comment:
This is not an image of cloud native development - it's just a picture of a cloud. I don't think it adds any value, but it does slow down the time it takes to load the page. Recommend removing.

Reply:
We like having some kind of engaging image at the beginning of these articles to break up the walls of text. Maybe we can find a different stock image.

Reply:
Will iterate on images as we iterate on the rest of the content.